requirements.txt

```diff
@@ -1,6 +1,6 @@
 mypy==0.660
 pytz==2019.2
 numpy==1.15.1
-pytest==4.1.1
+pytest==5.2.0
```

Comment:
Yes, I upgraded pytest version number because the previous one was giving me an error when running the tests (on Windows 10):

```
$ mingw32-make test
mypy --ignore-missing-imports mffpy
pytest --cov
Traceback (most recent call last):
  File "c:\users\damian\anaconda3\envs\mffpy\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\damian\anaconda3\envs\mffpy\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\Damian\Anaconda3\envs\mffpy\Scripts\pytest.exe\__main__.py", line 7, in <module>
  File "c:\users\damian\anaconda3\envs\mffpy\lib\site-packages\_pytest\config\__init__.py", line 61, in main
    config = _prepareconfig(args, plugins)
  File "c:\users\damian\anaconda3\envs\mffpy\lib\site-packages\_pytest\config\__init__.py", line 182, in _prepareconfig
    config = get_config()
  File "c:\users\damian\anaconda3\envs\mffpy\lib\site-packages\_pytest\config\__init__.py", line 156, in get_config
    pluginmanager.import_plugin(spec)
  File "c:\users\damian\anaconda3\envs\mffpy\lib\site-packages\_pytest\config\__init__.py", line 530, in import_plugin
    __import__(importspec)
  File "c:\users\damian\anaconda3\envs\mffpy\lib\site-packages\_pytest\tmpdir.py", line 25, in <module>
    class TempPathFactory(object):
  File "c:\users\damian\anaconda3\envs\mffpy\lib\site-packages\_pytest\tmpdir.py", line 35, in TempPathFactory
    lambda p: Path(os.path.abspath(six.text_type(p)))
TypeError: attrib() got an unexpected keyword argument 'convert'
makefile:10: recipe for target 'test' failed
mingw32-make: *** [test] Error 1
```

Then I did a quick search and I found [this](https://stackoverflow.com/a/58189684) answer suggesting that the best action is to update pytest to a more recent version.

All tests are passing on Windows 10 now.

Reply:
Sounds good man - we can consider this documented, but in the future when we update package requirements in `Requirements.txt`,  we should be in the habit of documenting why the change was made in the relevant commit.

Reply:
Will do!